doc/index.rst

```diff
@@ -0,0 +1,134 @@
+ ======================= ===== ====== ========= ======= ====== ====== ===== ===== ====== ======= ========
+
+For more information about the features of a specific peripheral please refer to the official documentation
+of the specific nPM device.
+
+How to use
+----------
+
+Repository initialization
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The recommended way to clone the repositories is by using the ``west`` tool. Information about the
+installation, available commands, configuration and more can be found in the
+`Zephyr's west`_ tool documentation.
+
+To initalize the npmx-zephyr repo, run the following commands:
+
+.. code:: bash
+
+   west init -m https://github.com/NordicSemiconductor/npmx-zephyr
+   cd npmx-zephyr
+   west update
+
+Alternatively, you can clone repositories by using ``git`` only.
+
+In this case pay special attention to revisions - both Zephyr and npmx revisions should match the
+ones specified in the ``west.yml``. Moreover ``npmx`` should be placed in ``zephyr/modules``
+subdirectory.
+
+Building and running sample application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Firstly, setup needs to be properly configured. Please read the documentation to learn how to do so.
+If you want to run one of the sample applications, follow the relevant instruction provided in the
+documentation for that specific sample.
+
+After configuring the setup, the project can be built and flashed to the target device.
+
+.. code:: bash
+
+   cd <path_to_npmx-zephyr>/samples/<desired_sample>
+   west build -b <target>
+   west flash
+
+For instance, if you want to run the LED sample for nPM1300 Engineering B used with nRF52840DK, you
+must configure the setup as specified in the :doc:`led sample description <../samples/led/README>`.
+
+.. code:: bash
+
+   cd npmx-zephyr/samples/led
+   west build -b nrf52840dk_nrf52840
+   west flash
+
+As a result, you should see LEDs blinking on the board.
+
+.. note::
+   The ``ZEPHYR_BASE`` variable must be set as the path to your Zephyr folder (the
+   one you have cloned with west).
+
+   You can set this variable manually with the following command:
+
+   .. code:: bash
+
+      export ZEPHYR_BASE=<path_to_zephyr_directory>
+
+Generating documentation
+------------------------
+
+All files required to compile the Sphinx-based documentation for ``npmx-zephyr`` are located under
+the ``doc`` folder.
+
+**Requirements**
+
+Install Python 3 and its dependencies with the following command:
+
+.. code:: bash
+
+   pip install -r scripts/requirements.txt
+
+**Build**
+
+You may want to use the provided script ``doc/sphinx_build_local.sh``.
+Open the ``doc/build/html/index.html`` file to see the output.
+
+If you want to build it manually, go to the ``doc`` directory and run the following commands:
+
+.. code:: bash
+
+   sphinx-build -M html . build
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   samples
+
+.. toctree::
+   :maxdepth: 1
+
+   additional_resources
+   release_notes
```

Comment:
```suggestion
.. toctree::
   :maxdepth: 1
   :caption: Contents:

   samples
   additional_resources
   release_notes
```

As end user have a full list of samples available on the left-side menu, let's have a short list at the end of this article, ok?

Reply:
The user doesn't have a full list on the left-side menu, one needs to click to see it.

I had a discussion with @masz-nordic, and we decided to make all samples visible at the end of index.rst. However, if you happen to know a way to have all samples listed on the left-side menu, I would appreciate it.